lib/DBIx/Class/ResultSource.pm

```diff
@@ -702,7 +702,7 @@ sub sequence {
 
 =over 4
 
-=item Arguments: $name?, \@colnames
+=item Arguments: $name?, \@colnames, \%extra?
 
```

Comment:
This is not a useful way to extend the signature - too many unknowns. Make it

`$name?, \%uc_info | \@colnames`

With both *names optional* and *\@colnames* marked as discouraged in the docs. I.e. people should be doing:
```
add_unique_constraints(
  foo => { ... }
  bar => { ... }
)
```

just like they do with `add_columns` today.